Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1158,6 +1158,14 @@ The current Cilium kube-proxy replacement mode can also be introspected through
     $ kubectl exec -it -n kube-system cilium-xxxxx -- cilium status | grep KubeProxyReplacement
     KubeProxyReplacement:   Strict	[eth0 (DR)]
 
+Graceful Termination
+*********************
+
+Cilium's eBPF kube-proxy replacement supports graceful termination of service
+endpoints deletion. The feature requires at least Kubernetes version 1.20, and
+the feature gate ``EndpointSliceTerminatingCondition`` needs to be enabled. The
+feature can be disabled with the configuration option ``enable-k8s-terminating-endpoint``.
```

Comment:
Could you add more verbosity in here? I think that would help users with the setup, e.g. what's the default time for the grace period? Also description on what happens in datapath during that period wrt terminating backend. Maybe also link to K8s docs, etc would help.